pinot-plugins/pinot-file-system/pinot-s3/src/test/java/org/apache/pinot/plugin/filesystem/S3PinotFSTest.java

```diff
@@ -287,7 +287,7 @@ public void testCopyFromAndToLocal()
 
     Assert.assertEquals(headObjectResponse.contentLength(), (Long) fileToCopy.length());
 
-    File fileToDownload = new File("copyFile_download.txt");
+    File fileToDownload = new File("copyFile_download.txt").getAbsoluteFile();
```

Comment:
Why do we change this in the test? This should be handled within the `PinotFS`

Reply:
because relative File.getParent will return null, which is not goingg to happen in the prod environment